src/frontend/screens/DownloadManager/index.tsx

```diff
@@ -84,81 +77,61 @@ export default React.memo(function DownloadManager(): JSX.Element | null {
+        </h3>
+        <Tabs
+          defaultValue="downloading"
+          classNames={getTabsClassNames(
+            { list: styles.tabsList, panel: styles.tabsPanel },
+            { list: 'outline' }
+          )}
+        >
+          <Tabs.List>
+            <Tabs.Tab value="downloading">Downloading</Tabs.Tab>
+            <Tabs.Tab value="queued">Queued</Tabs.Tab>
+            <Tabs.Tab value="downloaded">Downloaded</Tabs.Tab>
+          </Tabs.List>
+          <Tabs.Panel value="downloading">
+            <div className={styles.downloadManager}>
               <div
                 style={
                   !currentElement ? { backgroundColor: 'transparent' } : {}
                 }
-                className="downloadList"
+                className={styles.downloadList}
               >
-                <h5 className="downloadManagerCurrentSectionTitle">
-                  {t('queue.label.downloading', 'Downloading')}
-                </h5>
-                <div className="dmItemList">
-                  <DownloadManagerHeader time="started" />
-                  <DownloadManagerItem
-                    element={currentElement}
-                    current={true}
-                    state={state}
-                  />
-                </div>
+                <DownloadTable
+                  elements={currentElement ? [currentElement] : []}
+                  time="started"
+                />
               </div>
             </div>
-          )}
-          <h5 className="downloadManagerQueuedSectionTitle">
-            {t('queue.label.queued', 'Queued')}
-          </h5>
-          <div className="dmItemList">
-            <DownloadManagerHeader time="queued" />
-            {plannendElements.length > 0 ? (
-              plannendElements.map((el) => (
-                <DownloadManagerItem
-                  key={el.params.appName}
-                  element={el}
-                  current={false}
-                />
-              ))
-            ) : (
-              <DownloadManagerItem current={false} />
-            )}
-          </div>
-        </>
-      }
-      {!!doneElements?.length && (
-        <div className="downloadManager">
-          <div className="downloadList">
-            <span>
-              <h5 className="downloadManagerQueuedSectionTitle">
-                {t('queue.label.finished', 'Completed')}
-                <button
-                  className="button is-text"
-                  onClick={() => handleClearList()}
-                >
-                  {t('queue.label.clear', 'Clear List')}
-                </button>
-              </h5>
-            </span>
-            <div className="dmItemList">
-              <DownloadManagerHeader time="finished" />
-              {doneElements.map((el, key) => (
-                <DownloadManagerItem key={key} element={el} current={false} />
-              ))}
+          </Tabs.Panel>
+
+          <Tabs.Panel value="queued">
+            <>
```

Comment:
we can remove the `<></>` since there is only one child node

Reply:
nice catch